build-container/ccm.conf

```diff
@@ -0,0 +1,71 @@
+# Fully qualified path for build root.
+# This should not use a variable like $HOME.  If your machine has lots
+# of memory, consider locating this to tmpfs to avoid usage to the disk and
+# to minimize access times but know that unless you copy the contents to
+# physical media, it will not survive a reboot. See the manpage for tips.
+CHROOTPATH64="/scratch/.buildroot"
+
+# Number of threads makepkg in the clean chroot will use when building.
+# The typical rule is physical cores + 1.
+THREADS=9
+
+# Optionally uncomment and define a custom pacman.conf and/or a custom
+# makepkg.conf for the buildroot using a fully qualified path below.
+# Leaving these two undefined to use the system files.
+
+#CUSTOM_PACMAN_CONF='/usr/share/devtools/pacman.conf.d/extra-testing.conf'
+#CUSTOM_MAKEPKG_CONF='/usr/share/devtools/makepkg.conf.d/x86_64.conf'
+
+# Optionally uncomment and define a custom location and name for the local chroot
+# package repo.
+#REPO="/src/repo"
+REPO_NAME="chroot_local"
+
+# Optionally uncomment to pass the --nocheck flag to the build which will skip
+# the check function in the PKGBUILD is it is present.
+#NOCHECK=1
+
+# Optionally define the format of compression for compiled packages. Leave this
+# undefined to use the Arch default.
+PKGEXT=
+
+# If set, the value defined will be used in the buildroot's packages.
+PACKAGER=
```

Comment:
Is this set somewhere else? Where do we really set the PACKAGER entry for the packages?

Reply:
This config is taken from the current archzfs-ci (since that only appends a line to the bottom of this default config).

Looking at the [current published archzfs packages](https://zxcvfdsa.com/archzfs/archzfs/x86_64/), those also have `Unknown Packager` in their metadata.

We could set it to something like "archzfs-ci" maybe? Not sure.

Reply:
> Looking at the [current published archzfs packages](https://zxcvfdsa.com/archzfs/archzfs/x86_64/), those also have `Unknown Packager` in their metadata.

Funny that it seems to have been like this all these years :)

> We could set it to something like "archzfs-ci" maybe? Not sure.

Supposedly it should be "name/email of the person or organization building packages" (from makepkg.conf), but I wonder if we could use instead, e.g. `ZFS on Arch Linux (https://github.com/archzfs)`. Namcap will likely dislike it though. Let's maybe put just `ZFS on Arch Linux` for now and revisit it after merging.

Reply:
Let's set this to `ArchZFS Project (https://github.com/archzfs)`, perhaps? It's what I set as name in the PGP key. Not ideal perhaps, but if needed we can revisit it later.